### ISSUE TITLE
Is there any possibility to extend the project as stable diffusion extension

### ISSUE THREAD

**Issue by kaninbu** [XingangPan/DragGAN#26]
单纯问个, 大佬有打算把它整合进stable diffusion吗？

这对于许多人来说应该会有很大的帮助，尤其是在控制stable diffusion生成奇怪的图之下用于控制然后输出成自己想要的样子

**Comment by andrewbutts** [comment]
Is it easy enuf to optimize a stable diffusion latent image instead of a gan latent code?  Seems like it'd be possible but maybe a lot slower.

**Comment by PDillis** [comment]
No. That is another paper by itself, as you'll have to focus entirely on the StableDiffusion architecture (DragGAN uses only the inner representations of the Generator of StyleGAN2). 

<img width="587" alt="image" src="https://github.com/XingangPan/DragGAN/assets/24496178/dc29a57b-124d-4c19-8e25-b57c178149bc">

It is a good idea, but isn't as straightforward as in GANs, so perhaps you will need to focus on the neighborhood of the synthesized images during the diffusion process in the image space as in [Align Your Latents](https://research.nvidia.com/labs/toronto-ai/VideoLDM/).

**Comment by LianShuaiLong** [comment]
两种不同的生成模型，无法整合的

**Comment by PDillis** [comment]
There is now a project inspired by this, though I have no idea when they'll release code or how well it works (it's a work in progress after all): https://arxiv.org/abs/2306.14435

![image](https://github.com/XingangPan/DragGAN/assets/24496178/773cf7db-e1cd-4bf6-9b69-57be20d11901)

**Comment by zjysteven** [comment]
> There is now a project inspired by this, though I have no idea when they'll release code or how well it works (it's a work in progress after all): https://arxiv.org/abs/2306.14435
> 
> ![image](https://user-images.githubusercontent.com/24496178/249175271-773cf7db-e1cd-4bf6-9b69-57be20d11901.png)

Not trying to be destructive in any sense, but it seems from this figure that DragDiffusion does not show clear advantage over DragGAN (most illustrated movements/changes are kinda small). I wouldn't be surprised if this is indeed the case though as I'd imagine dragging the latents of diffusion model would be more difficult given its forward/diffusion process.
